### PR TITLE
A couple of things we missed

### PR DESCRIPTION
**Pull request by guroseter**
Put all components into the components package. Rewrote the Game component to stateless (so we don't have to explain render, constructor, extends component).
Implemented ResetGame component. 
Set state of the prop finished to false on GAME_RESET action.

### CODE REVIEW

File: day3/src/reducers.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const cardReducer = (state = initialStateCards, action) => {
 };
 
 const initialStateGame = {
-  finished: false,
+  finished: true,
```
Comment (ingara) [inline day3/src/reducers.js]: Spillet skal vel ikke være `finished` når man starter?

Comment (guroseter) [inline day3/src/reducers.js]: Oups, jeg satt bare den for å teste. Orket ikke å spille hele spillet :P Bra catch! :-D

Comment (ingara) [inline day3/src/reducers.js]: Pft, det bør nå sitte godt i fingrene snart.

**Comment by ingara** [comment]
Flotte greier! :+1:

**Pull request merged**